specs/disciplina/educator/api/student.yaml

```diff
@@ -0,0 +1,414 @@
+      produces: 
+        - application/json
+      responses:
+        200:
+          description: successful operation
+          schema:
+            $ref: '#/definitions/Course'
+        404:
+          description: Course with given ID not found
+  
+  /assignments:
+    get:
+      tags:
+        - assignments
+      summary: Get student\'s assignments
+      description: Gets a list of student\'s assignments. Filter parameters are used to specify
+        specific course, type, etc.
+      operationId: getAssignments
+      parameters: 
+        - $ref: '#/parameters/CourseId'
+        - $ref: '#/parameters/DocumentType'
+        - in: query
+          name: final
+          required: false
+          description: Select only final assignments
+          type: boolean
+      produces:
+        - application/json
+      responses:
+        200:
+          description: successful operation
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/Assignment'
+
+  /assignments/{assignmentHash}:
+    get:
+      tags:
+        - assignments
+      summary: Get info about an assignment
+      description: Gets an assignment info by given submission hash. Returns 404 if a student tries to get an assignment which is not assigned to them.
+      operationId: getAsssignment
+      parameters:
+        - $ref: '#/parameters/AssignmentHashPath'
+      produces:
+        - application/json
+      responses:
+        200:
+          description: successful operation
+          schema:
+            $ref: '#/definitions/Assignment'
+        404:
+          description: Assignment with given wasn not found (or a user has no rights to look it up)
+
+  /submissions:
+    get:
+      tags:
+        - submissions
+      summary: Get student\'s submissions
+      description: Gets a list of student\'s submissions. Filter parameters are used to specify
+        specific course, assignment, etc.
+      operationId: getSubmissions
+      parameters: 
+        - $ref: '#/parameters/CourseId'
+        - $ref: '#/parameters/AssignmentHash'
+        - $ref: '#/parameters/DocumentType'
+      produces:
+        - application/json
+      responses:
+        '200':
+          description: successful operation
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/Submission'
+    post:
+      tags:
+        - submissions
+      summary: Make a new submission
+      description: Posts a new submission with a given body. Request body should contain valid student\'s signature of submission contents, otherwise an error will be raised.
+      operationId: makeSubmission
+      produces:
+        - application/json
+      consumes:
+        - application/json
+      parameters:
+        - in: body
+          name: submission
+          description: A submission
+          required: true
+          schema:
+            $ref: '#/definitions/Submission'
```

Comment:
`Submission` has "grade" field. I doubt that the student must send any grades to the educator. He should send the "ungraded submission" instead.

EDIT: "grade" field of the submission is not required, I see. The automatic example in swagger is just bad.